log/standardLogger.go

```diff
@@ -65,10 +74,6 @@ func (jf jsonFormat) Format(entry *logrus.Entry) ([]byte, error) {
 func NewStandardLogger() Logger {
 	logger := logrus.New()
 	logger.SetFormatter(&standardFormat{})
-
-	// makes sure that it always logs every level
-	logger.SetLevel(logrus.DebugLevel)
-
 	// explicitly set it to os.Stderr
 	logger.SetOutput(os.Stderr)
```

Comment:
Why?

Reply:
thought that was removed, will delete

Reply:
So that by default, standardLogger has its verbosity set to off

Reply:
I was asking about setting output to `Stderr`.

Reply:
Oh sorry. I think it was added for an error I had in the first iteration of the library. I'll remove it.